modules/nf-core/gatk4/baserecalibratorspark/main.nf

```diff
@@ -3,7 +3,7 @@ process GATK4_BASERECALIBRATOR_SPARK {
     label 'process_low'
 
     conda "bioconda::gatk4=4.4.0.0 conda-forge::openjdk=8.0.312"
-    container 'docker.io/broadinstitute/gatk:4.4.0.0'
+    container "nf-core/gatk:4.4.0.0"
```

Comment:
Why did we mirror all the containers?

Reply:
so that we can change the registry for all container in a single line